### ISSUE TITLE
Free d6 attribute in character builder

### ISSUE THREAD

**Issue by jasharen** [kristianserrano/swade-item-tables-issue-tracker#12]
Am I missing something in the character builder. When my players and I select Human (they get a free d6 stat), and I go to assign attributes, I only have 5 attribute points to spend. If I bump my first stat to a d6, it uses one of my stat allocations. I'd expect a free d6 to be reflected by having 6 stat assignments. We currently get around it by adding a free stat boost in the field supplied. But this seems like a work around. 

Is this a future or missing feature? Or are we doing something wrong? Amazing work!

**Comment by kristianserrano** [comment]
Thanks for submitting this issue. I'm not sure which version of a Human has the free d6 stat, and the only version I know of specifies that it's a skill.

Either way, the answer is "it depends". It's entirely possible for an Ancestry to have an Active Effect that increases a Trait by 2 die sides (i.e., one die type), in which case, the Character Creator does account for that (it looks at the source data rather than the derived data). In cases where that's just a text description in the ancestry, the Character Creator does not automatically account for it. That's where the free number field can help as a workaround.

Ideally though, the Ancestry would grant an ability that carries a choice set that allows you to pick a Trait which would then apply the appropriate Active Effect.

Does the Human ancestry your using have such an Active Effect on it?

**Comment by jasharen** [comment]
Savage worlds pathfinder:

Adaptability: Humans start with a free Edge (they must meet the requirements, as usual). They also gain a d6 in any one Attribute instead of a d4. This does not increase the attribute limit.

**Comment by jasharen** [comment]
If I click the "has adaptability" check it does give me the edge, but even though the Human 'item' has the Adaptability item grant I still have to check off to get the free edge. Either way however, the free d6 does not work without using the adjust attribute point.

**Comment by jasharen** [comment]
In savage worlds pathfinder there are a bunch of races that get the free d6. Halflings jump to mind immediately. Where do I go to submit a ticket to Metamorphic? Thanks for everything, the add on has been immensely useful in helping my group adapt to Savage Worlds.

**Comment by kristianserrano** [comment]
> In savage worlds pathfinder there are a bunch of races that get the free d6. Halflings jump to mind immediately. Where do I go to submit a ticket to Metamorphic? Thanks for everything, the add on has been immensely useful in helping my group adapt to Savage Worlds.

Yeah, I have a few of those in Savage Eberron, and I use Active Effects to handle them.

You can submit issues and requests through their support portal at https://support.metamorphic-digital.com/
